presto-docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -0,0 +1,25 @@
+====================
+Black Hole Connector
+====================
+
+The Black Hole connector works in similar way as /dev/null device file in Linux operating system. 
```

Comment:
+SELECT statement from any of its tables will return no rows and INSERT will also be mocked and no real
+writes will be performed.

+All information (meta data) about created tables live only in memory, so when you restart the server they
+will be discarded.

Insert data into table in blackhole connector::